### ISSUE TITLE
NeMo's BERT based intent slot classification model (nemo_nlp.models.IntentSlotClassificationModel) fails to train

### ISSUE THREAD

**Issue by vivekrkayq** [NVIDIA/NeMo#5459]
NeMo's BERT based intent slot classification model (nemo_nlp.models.IntentSlotClassificationModel) fails to train (training loss remains the same after every epoch) when we use NeMo's stable versions (eg. v1.12.0, 'main' branch). However it trained perfectly well when we used the `r1.11.0` branch instead.

Please run this [Joint Intent Slot Classification](https://github.com/NVIDIA/NeMo/blob/main/tutorials/nlp/Joint_Intent_and_Slot_Classification.ipynb) notebook as it is to reproduce the bug.

 - Environment location: GCP Compute Engine (NVIDIA T4 Tensor Core GPU)
 - Method of NeMo install: 
       `BRANCH = 'main'`
       `!python -m pip install git+https://github.com/NVIDIA/NeMo.git@$BRANCH#egg=nemo_toolkit[nlp]`

**Comment by clenki** [comment]
I can confirm this bug. Tried it on docker, ubuntu and windows with my own data and the example data. The models trained with BERT intent slot classification seems to "freeze" on their initial loss and val_loss values. They are not doing anything while training. I tried to use r1.11.0 branch like @vivekrkayq mentioned but this failed for me aswell. Last try was with:
nemo-toolkit 1.11.0, pytorch-lightning 1.7.6, torch 1.13.0, nvidia driver: 515.43.04 with cuda 11.7

@vivekrkayq Would you be so kind to post your pip list? Maybe it can help me to get it running as I need it.
